qutip/qip.py

```diff
@@ -0,0 +1,13 @@
+"""Module replicating the qutip_qip package from within qutip."""
+import sys
+
+try:
+    import qutip_qip
+
+    sys.modules["qutip.qip"] = sys.modules["qutip_qip"]
+    del qutip_qip
```

Comment:
```suggestion
    import qutip_qip
    del qutip_qip
    sys.modules["qutip.qip"] = sys.modules["qutip_qip"]
```
You can actually `del qutip_qip` staight away if you're going to look it up in sys.modules anyway.

Reply:
True, will do